### PR TITLE
New goryman (riemann) lib location

### PR DESCRIPTION
**Pull request by rikatz**
TL;DR

Old goryman library location (github.com/bigdatadev/goryman) does not exists anymore. 

This is breaking any godep operation. I've forked an already goryman forked repo to my own repo (github.com/rikatz/goryman), rebased the import from inside the library (https://github.com/rikatz/goryman/commit/8c8d07c1fe9b7331498287804b68a17cd66be8bd) so this library is 'usable' again.

And this PR points to this modified library.

The checksum from the files contained on 'vendor/github.com/bigdatadev/goryman' and on the forked repository matches (they're the same version). The only change is contained on the commit above.

### CODE REVIEW

**Comment by rikatz** [comment]
@DirectXMan12 this PR is about the Godeps problem I've mentioned in #1443

**Comment by DirectXMan12** [comment]
This fixes issue #1419 (which has another repo location posted).  I don't particularly care which repo location we use, but I think we're probably going to have to deprecate (then remove) the riemann sink.

**Comment by DirectXMan12** [comment]
cc @jsoriano

**Comment by jsoriano** [comment]
@DirectXMan12 LGTM, thanks @rikatz !

**Comment by jsoriano** [comment]
@rikatz you could maybe squash the three changesets into one

**Comment by rikatz** [comment]
@jsoriano Don't know if I did right, but tried to squash them in one commit.

**Comment by jsoriano** [comment]
@rikatz umm, there are 5 changesets now, take a look to this http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

**Comment by jsoriano** [comment]
@rikatz it seems that you did it right as everything seems to be in 21248d80dfd8762a38f4805182f80a12c50342ab but you merged before pushing instead of forcing the push, this should do the trick:

```
git reset --hard 21248d80dfd8762a38f4805182f80a12c50342ab
git push -f
```

**Comment by rikatz** [comment]
@jsoriano Made a real mess here. Thinking if it's better to erase everything and open a new PR.

**Comment by rikatz** [comment]
OK, I give up! Will open a newer, fresher and cleaner one.

**Comment by jsoriano** [comment]
@rikatz but no new PR is needed, 21248d80dfd8762a38f4805182f80a12c50342ab was fine, just go to your riemmanlib branch and run these two commands, this will be ok :)

**Comment by rikatz** [comment]
@jsoriano Tried again and failed in a miseable way :( New PR seems easier to me, a git dumb user :)

**Comment by rikatz** [comment]
(this is what happens when Ops try to be Devs :P )

**Comment by rikatz** [comment]
@DirectXMan12 actually I don't care also about deprecating this sink (I'm a ES user anyway), but there is an option of changing the lib from this gorymann to this one: https://github.com/amir/raidman

I don't know how hard is to do this (and if even does matter to anyone), but this is an option.

**Comment by jsoriano** [comment]
If we finally keep the riemman sinks I could take a look to this lib. It seems to have a quite similar interface.
But while we decide what to do I think that the best option is to merge your PR.
